.NET/Microsoft.Recognizers.Text.DateTime/German/Extractors/GermanTimeExtractorConfiguration.cs

```diff
@@ -9,11 +9,13 @@ public class GermanTimeExtractorConfiguration : BaseOptionsConfiguration, ITimeE
     {
         // part 1: smallest component
         // --------------------------------------
-        public static readonly Regex DescRegex = new Regex(DateTimeDefinitions.DescRegex,
+        public static readonly Regex DescRegex = new Regex(
```

Comment:
Nit: specifically for regexes, this class is not really following the same convention as the others, right?

Reply:
We updated to single line parameters instead.